test/fixtures/payout_fixtures.js

```diff
@@ -4,10 +4,10 @@ const pact = pactBase()
 
 function buildPayout (opts) {
   return {
-    gateway_payout_id: 'some-gateway-payout-id',
+    gateway_payout_id: opts.gateway_payout_id || 'some-gateway-payout-id',
```

Comment:
It would be good to keep the options for this builder consistent, it looks like at the moment they use camelCase (`paidOutDate`). 

I think it's fine to use either but if we're changing to snake_case it should be the same for the other params